archinstall/lib/installer.py

```diff
@@ -248,9 +248,15 @@ def set_timezone(self, zone, *args, **kwargs):
 
 	def activate_ntp(self):
 		self.log('Installing and activating NTP.', level=logging.INFO)
```

Comment:
This log line should be changed to "Configuring systemd-timesyncd for NTP functionality" or similar.

Reply:
I'll probably chance in guided as well to better reflect this, and rename the function name, good call!